datadog/resource_datadog_synthetics_test_.go

```diff
@@ -1075,6 +1075,11 @@ func syntheticsConfigVariable() *schema.Schema {
 					Type:        schema.TypeString,
 					Optional:    true,
 				},
+				"secure": {
+					Description: "Whether the value of this variable will be obfuscated in test results.",
+					Type:        schema.TypeBool,
+					Optional:    true,
```

Comment:
Lets set default to false explicitly since we can't differentiate between zero values and unset values in tf.
```suggestion
					Optional:    true,
					Default:     false,
```